topydo/ui/Main.py

```diff
@@ -191,17 +203,32 @@ def _execute_handler(self, p_command, p_output=None):
             # TODO: show error message
             pass
 
-    def _update_all_columns(self):
+    def _update_all_columns(self, p_reset_keystate=False):
```

Comment:
Is the new parameter really required? What could break if we reset it unconditionally?